docs/faq/contributing.txt

```diff
@@ -84,16 +84,16 @@ of a larger problem. While we can spend time writing, testing and applying
 lots of little patches, sometimes the right solution is to rebuild. If a
 rebuild or refactor of a particular component has been proposed or is
 underway, you may find that bugs affecting that component will not get as much
-attention. Again, this is just a matter of prioritizing scarce resources. By
+attention. Again, this is a matter of prioritizing scarce resources. By
 concentrating on the rebuild, we can close all the little bugs at once, and
 hopefully prevent other little bugs from appearing in the future.
 
 Whatever the reason, please keep in mind that while you may hit a particular
-bug regularly, it doesn't necessarily follow that every single Django user
-will hit the same bug. Different users use Django in different ways, stressing
+bug regularly, it doesn't necessarily follow that every single Django user will
+hit the same bug. Different users use Django in different ways, stressing
```

Comment:
This patch is quite big, can you revert all unrelated changes like this :pray:

Reply:
Hmm, I rewrapped all text I changed to 80 characters, because I thought docs were supposed to have consistent wrapping, but yeah, I can try to revert those.

Reply:
Documentation should be wrapped at 79 chars (see [#python-style](https://docs.djangoproject.com/en/2.2/internals/contributing/writing-code/coding-style/#python-style)). But even if it's not wrapped in some places, please don’t waste time doing unrelated refactoring of existing documentation to conform to this style.

Reply:
This will leave a couple of weird short lines in the middle of text, but I guess it's preferable for reviewing purpose. Sorry for the noise.

Reply:
So to be clear if you for example remove word `easy` from some paragraph then yes we should fix wrapping for the entire paragraph at 79 chars. I'm only talking about omitting totally unrelated changes like with this `will`.

Reply:
I think I made some changes to that paragraph and reverted them, and by accident the wrapping stayed in. I tried to touch only the paragraphs I changed. Going to revert this change.

Reply:
I went through the patch and made sure no further unrelated changes were made, and changed the wrapping to change paragraphs I touched only from the point of change, not above that.